ui/src/contexts/Application.jsx

```diff
@@ -70,8 +70,9 @@ function watchVault(id, dispatch, offerStatus) {
   }
 
   async function vaultUpdater() {
-    const uiNotifier = E(walletP).getUINotifier(id);
-    for await (const value of iterateNotifier(uiNotifier)) {
+    // TODO: Do something with asset notifier.
```

Comment:
without the asset notifier the UI can't show the actual debt. It has to be calculated from the vault's debtSnapshot and the manager/asset's current compounded interest. Like so https://github.com/Agoric/agoric-sdk/blob/52f60eb/packages/run-protocol/src/vaultFactory/vault.js#L218-L223

Is that in scope for this PR?

Reply:
I actually wasn't sure how that was supposed to work, thanks. I think it's worth merging this first since that seems like a separate step, and the dapp crashes without this current PR as is. I can create a separate issue for that.